setup.py

```diff
@@ -9,7 +9,9 @@
     license='',
     install_requires=['thrift==0.9.2',
                       'opentracing==1.0rc3',
-                      'jsonpickle'],
+                      'jsonpickle',
+                      'basictracer==1.0rc1',
+                      'protobuf>=3.0.0b2.post2'],
```

Comment:
is there a direct dependency, or just the indirect dep through basictracer? if the latter (which I think is the case), I think/hope we can remove this line here.